tests/autokeras/hypermodel/preprocessor_test.py

```diff
@@ -72,12 +73,27 @@ def test_ngram():
              'The dog sat on the log.',
              'Dogs and cats living together.']
     dataset = tf.data.Dataset.from_tensor_slices(texts)
+    # Test Default Behaviour
     new_dataset = run_preprocessor(
         preprocessor_module.TextToNgramVector(),
         dataset,
         common.generate_data(dtype='dataset'),
         tf.float32)
     assert isinstance(new_dataset, tf.data.Dataset)
+    # Test ngram_range
+    new_dataset = run_preprocessor(
+        preprocessor_module.TextToNgramVector(ngram_range=(1, 2)),
+        dataset,
+        common.generate_data(dtype='dataset'),
+        tf.float32)
+    assert isinstance(new_dataset, tf.data.Dataset)
+    # Test stop word removal
+    new_dataset = run_preprocessor(
+        preprocessor_module.TextToNgramVector(stop_words=ENGLISH_STOP_WORDS),
+        dataset,
+        common.generate_data(dtype='dataset'),
+        tf.float32)
+    assert isinstance(new_dataset, tf.data.Dataset)
```

Comment:
We should separate these assertions run_preprocessor calls to different test functions according to the single path rule of unittesting. Please add one more test to verify the output of your implementation is almost the same as the output of the original sklearn method.